### PR TITLE
chore: add lavakush

### PR DESCRIPTION
**Pull request by lavakush07**
Added my file to Linkfree

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/986"></a>

### CODE REVIEW

**Review by emmalearnscode — APPROVED** [review]

File: public/data/Lavakush Biyani.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/lavkushbiyani1",
+      "icon": "twitter"
+    },
+    {
+     "name":"Follow me on Linkedin",
+      "url":"https://www.linkedin.com/in/lavkush-biyani-07/",
+      "icon": "Linkedin"
```
Comment (Panquesito7) [inline public/data/Lavakush Biyani.json:20]:
```suggestion
      "icon": "linkedin"
```

File: public/data/Lavakush Biyani.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      "date": "April 2021",
+      "icon": "code",
+      "color": "grey",
+      "description": "Started building websites"
+    },{
+      "title":"Started competitive programming",
+      "date":"December 2020",
+      "icon":"code",
+      "color":"blue",
+      "description":"Started participating on codechef,Geeks for Geeks"
```
Comment (Panquesito7) [inline public/data/Lavakush Biyani.json:35]: I think it'd look better like this 😅 What do you think? 
```suggestion
      "description":"Started participating on CodeChef, Geeks for Geeks"
```

File: public/data/Lavakush Biyani.json [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+      "url": "https://github.com/lavakush07",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/lavkushbiyani1",
+      "icon": "twitter"
+    },
+    {
+     "name":"Follow me on Linkedin",
```
Comment (Panquesito7) [inline public/data/Lavakush Biyani.json:18]: I think we should capitalize the `i` 😅 What do you think? 
```suggestion
     "name":"Follow me on LinkedIn",
```

File: public/data/Lavakush Biyani.json (hunk 1 above)
Comment (emmalearnscode) [reply public/data/Lavakush Biyani.json:20]: @Panquesito7 it doesn't matter if this is uppercase or lowercase, we have a function in the code to deal with this :)

Comment (emmalearnscode) [reply public/data/Lavakush Biyani.json:20]: It's still there, it's in the ProfilePage.js component

File: public/data/Lavakush Biyani.json (hunk 3 above)
Comment (lavakush07) [reply public/data/Lavakush Biyani.json:18]: Haha 😀, Do I  commit the changes u made it ? I am in confusion 😅

Comment (Panquesito7) [reply public/data/Lavakush Biyani.json:18]: > Haha 😀, Do I commit the changes u made it ? I am in confusion 😅

Sure if you want, except for the resolved conversation. :)

**Review by RonenTGreat — APPROVED** [review]

**Review by krishguptadev — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
LGTM 🎉, thank you for contributing to LinkFree

**Review by Panquesito7 — APPROVED** [review]

**Comment by lavakush07** [comment]
Why was my profile is not appearing on eddiehub linkfree page ?

**Comment by emmalearnscode** [comment]
@lavakush07 adding profiles requires us to manually deploy the app after it’s been merged and sometimes we forget to do that step. Have deployed now so your profile should appear soon 🙂

**Comment by lavakush07** [comment]
Ohhh I got it. Thank u @emmalearnscode

**Pull request merged**